packages/office-ui-fabric-react/src/components/Button/CompoundButton/CompoundButton.styles.ts

```diff
@@ -41,17 +47,26 @@ export const getStyles = memoizeFunction((
 
     label: {
       margin: '0 0 5px',
-      lineHeight: '100%'
+      lineHeight: '100%',
+      fontWeight: FontWeights.semibold
     },
-
     description: [
       theme.fonts.small,
       {
-        color: theme.palette.neutralSecondary,
         lineHeight: '100%'
       }
     ],
 
+  };
+
+  let standardCompoundTheme: IButtonStyles = {
+
+    ...standardStyles(theme),
```

Comment:
mixing things in like this is not recommended. For example, if standardStyles has a description object, the definition below would get overridden!